django-server/server/home/templates/home/home.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="PL_pl">
+
+<head>
+    <meta charset="utf-8">
+    <title>{% block title %}EatersLab Home{% endblock %}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="Eaters Lab Home Page"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+</head>
+
+
+<body>
+    <div id="content" class="container">
+        {% block content %}<h1>Welcome to EatersLab made by PoohLab.</h1>{% endblock %}
+        {% csrf_token %}
```

Comment:
CSRF Token used for testing

Reply:
should it be left for offical release page?

Reply:
For now I see no problem, https encrypt it, and it's not like you need to re-generate it every time